src/components/CreateAdministration.vue

```diff
@@ -24,10 +24,10 @@
               <PvInputText
                 id="administration-public-name"
                 v-model="state.administrationPublicName"
-                style="width: 20vh"
+                style="width: 50vh"
                 data-cy="input-administration-name"
               />
-              <label for="administration-public-name" class="w-full">Public Administration Name</label>
+              <label for="administration-public-name" style="width: 50vh">Public Administration Name</label>
```

Comment:
If we implement the section above, then this change can be reverted back to how it was.
```suggestion
              <label for="administration-public-name" class="w-full">Public Administration Name</label>
```